tests/container_test.py

```diff
@@ -26,33 +19,36 @@ def test_container_count(dockerc):
     ), "Wrong number of containers were started."
 
 
-def test_wait_for_ready(main_container):
-    """Wait for container to be ready."""
-    TIMEOUT = 10
-    for i in range(TIMEOUT):
-        if READY_MESSAGE in main_container.logs().decode("utf-8"):
-            break
-        time.sleep(1)
-    else:
-        raise Exception(
-            f"Container does not seem ready.  "
-            f'Expected "{READY_MESSAGE}" in the log within {TIMEOUT} seconds.'
-        )
+# def test_wait_for_ready(main_container):
+#     """Wait for container to be ready."""
+#     TIMEOUT = 10
+#     for i in range(TIMEOUT):
+#         if READY_MESSAGE in main_container.logs().decode("utf-8"):
+#             break
+#         time.sleep(1)
+#     else:
+#         raise Exception(
+#             f"Container does not seem ready.  "
+#             f'Expected "{READY_MESSAGE}" in the log within {TIMEOUT} seconds.'
+#         )
```

Comment:
Are you planning to implement something similar to these tests?  If so, there should be a TODO and an issue.

Reply:
Please see commit f6f77d0.